### PR TITLE
Updating code as per GitHub issue

### PR DESCRIPTION
**Pull request by SnehaGunda**
GitHUb issue- https://github.com/MicrosoftDocs/azure-docs/issues/6905

## Purpose
A uaer reported that existing code doesn't support
"mongodb": "^3.0.5",
"mongoose": "^5.0.13",
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by ydogandjiev — COMMENTED** [review]
I tried out this change but it doesn't work with the version of mongodb and mongoose specified in the package.json. I think you will also need to update to the latest version of mongodb and mongoose. I updated them to the following in your branch and app worked great:
"mongodb": "^3.0.6",
"mongoose": "^5.0.14",

File: server/mongo.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const env = require('./env/environment');
 
 mongoose.Promise = global.Promise;
```
Comment (ydogandjiev) [inline server/mongo.js:5]: @SnehaGunda, are you sure this change is necessary? I didn't find it to be needed when building and running the code.

**Review by ydogandjiev — COMMENTED** [review]
@SnehaGunda, you might want to pull in @burkeholland to help out with this change since he is probably most familiar with the code.

File: server/mongo.js (hunk 1 above)
Comment (SnehaGunda) [reply server/mongo.js:5]: Hey yes its Burke's suggestion, I had an email conversation with him

Comment (SnehaGunda) [reply server/mongo.js:5]: I am closing this PR, seems like I didn't open it to the right target, will create a new one

**Pull request closed without merging**